deeplearning4j-scaleout/deeplearning4j-scaleout-parallelwrapper/src/main/java/org/deeplearning4j/parallelism/trainer/DefaultTrainer.java

```diff
@@ -56,7 +56,7 @@
     @Builder.Default protected volatile boolean useMDS = false;
     protected final String uuid = UUID.randomUUID().toString();
     @Builder.Default protected boolean onRootModel = false;
-    @Builder.Default protected AtomicLong lastEtlTime = new AtomicLong(0);
+    @Builder.Default protected volatile AtomicLong lastEtlTime = new AtomicLong(0);
```

Comment:
Ye, that was me looking for some race condition in standalone PW, and just tried to be sure that's not the case.